package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "safeapp",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
This should be at least a minor version bump (not just a patch) since this change affects how this extension creates promoted posts in the CMS (rather than just changing internal details)